editors/code/src/config.ts

```diff
@@ -213,12 +213,14 @@ export class Config {
 
     get serverExtraEnv(): Env {
         const extraEnv =
-            this.get<{ [key: string]: string | number } | null>("server.extraEnv") ?? {};
+            this.get<{ [key: string]: { toString(): string } | undefined | null } | null>(
+                "server.extraEnv",
+            ) ?? {};
         return substituteVariablesInEnv(
             Object.fromEntries(
                 Object.entries(extraEnv).map(([k, v]) => [
                     k,
-                    typeof v !== "string" ? v.toString() : v,
+                    v ? (typeof v !== "string" ? v.toString() : v) : null,
```

Comment:
i don’t think this is right, consider for example `”RUST_TEST_THREADS”: 0`

Reply:
That should set it `RUST_TEST_THREADS` to `0` which it still does no?

Reply:
Ah, nevermind I see the issue ugh. I hate these languages